File-Formats/DocIO/word-to-image.md

```diff
@@ -332,6 +332,319 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-Image-conversion/Custom-image-resolution).
 
+## Fallback fonts
+
+During Word to Image conversions, if a glyph of the input text is unavailable in the specified font, the text will not be rendered properly. To address this, the Syncfusion Word (DocIO) library allows users to specify fallback fonts. When a glyph is missing, the library will use one of the fallback fonts to render the text correctly in the output image.
+
+Users can configure fallback fonts in the following ways:
+* Initialize default fallback fonts.
+* Set custom fonts as fallback fonts for specific script types, including Arabic, Hebrew, Chinese, and Japanese etc.
+* Set custom fonts as fallback fonts for a particular range of Unicode text.
+
+N> DocIO internally uses user initialized or specified fallback fonts for Unicode characters during Word to Image conversion. Therefore, the specified fallback fonts must be installed in the production environment or embedded in the input Word document (DOCX). Otherwise, it will not render the text properly using the fallback fonts.
+
+### Initialize default fallback fonts
+
+The following code example demonstrates how to initialize a default fallback fonts while converting a Word document to Image. The *InitializeDefault* API sets the default fallback fonts for specific script types like Arabic, Hebrew, Chinese, Japanese etc.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+//Opens the file as stream
+using (FileStream inputStream = new FileStream("Template.docx", FileMode.Open, FileAccess.Read))
+{
+   //Loads an existing Word document file stream
+   using (WordDocument wordDocument = new WordDocument(inputStream, Syncfusion.DocIO.FormatType.Docx))
+   {
+      //Initialize the default fallback fonts collection.
+      wordDocument.FontSettings.FallbackFonts.InitializeDefault();
+      //Instantiation of DocIORenderer for Word to PDF conversion.
+      using (DocIORenderer render = new DocIORenderer())
+      {
+         //Convert the entire Word document to images.
+         Stream[] imageStreams = wordDocument.RenderAsImages(); 
+         int i = 0;
+         foreach (Stream stream in imageStreams)
+         {
+             //Reset the stream position.
+             stream.Position = 0;
+             //Save the stream as file.
+             using (FileStream fileStreamOutput = File.Create("WordToImage_" + i + ".jpeg"))
+		     {
+		         stream.CopyTo(fileStreamOutput);
+		     }
+		     i++;
+		 }
+      }
+   }
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+### Fallback fonts based on script type
+
+The following code example demonstrates how a user can add fallback fonts based on the script types, which DocIO considers internally when converting a Word document to Image.
```

Comment:
The following code example demonstrates how a user can add fallback fonts based on the script types, which DocIO considers internally when converting a Word document to an Image.

Reply:
Modified the sentences.